docs/self-managed/identity/user-guide/managing-user-access.md

```diff
@@ -0,0 +1,45 @@
+---
+id: managing-user-access
+title: "Managing user access to Identity"
+sidebar_label: "Managing user access"
+---
+
+In this guide, you will learn how to grant users different levels of access to the Identity application.
+
+## Permissions supported by Identity
+
+Identity implements the following permissions:
+
+- `read`: Users can access all pages in Identity. They _cannot_ create, modify, or delete any data.
+- `read:users`: Users can access only the **Users** page and related subpages.
+- `write`: Users have access to all pages. They can create, modify, and delete data.
+
+:::note
+If you are connecting Identity to your existing Keycloak installation, the `read:users` permission needs to be created
+manually in Keycloak.
+:::
+
+You can [assign the above permissions to users as part of a role](assigning-a-role-to-a-user.md).
+This gives the user access to the Identity application.
+
+## Assign Identity permissions to a user
+
+Without an Identity permission, users can access Identity only to log in to other applications.
+They are not able to access the Identity UI beyond the login page.
+
+To grant a user access to the UI, you need to assign at least one Identity permission as part of a role to the user. This can be achieved in one of the following ways:
+
+### Using Keycloak presets
```

Comment:
Are these the component presets?

Reply:
In general, yes that was the intention. I'm not sure about the `Identity` role though. Would you consider it part of the component presets? 

I realise that the title and formulations in the text might be too vague. Perhaps a better way would be to write something like "Using our component presets" and "When you start Identity with our pre-configured Keycloak container...". Wdyt? We could also link the docs section on initialisation.

Reply:
For me Identity is separate to the component presets. Your alternative suggestions here sound good to me, though in the future even with a customer owned Keycloak the Identity role will automatically be created. I'd be happy with your suggestions now and we can always adjust if necessary :)